src/scripts/crowdin/reports/reportsHelpers.ts

```diff
@@ -25,22 +24,45 @@ export async function fetchTranslationCostsReport(
 
   // Todo: Remove ts-ignore when this PR gets merged
```

Comment:
time to ✂️ this comment?

Reply:
Looks like it... yeah I also see the commented code line below that I can patch, thanks!